apps/02RockPaperScissors.js

```diff
@@ -7,10 +7,43 @@ var rl = readline.createInterface({
   output: process.stdout
 });
 
+  // Hand one is a variable that will just have the word Rock Paper or Scissor in it!
+
+  //Spec 1 - If it's a tie, return "It's a tie!" It must be exactly "It's a tie!" for the tests to pass
+
 
 function rockPaperScissors(hand1, hand2) {
+  if (hand1.toLowerCase() === hand2.toLowerCase()) {
+    return "It's a tie!";
+  }
+
+  if (hand1.toLowerCase() === 'rock') {
+    if (hand2.toLowerCase() === 'scissors') {
+      return 'Hand one wins!';
+    }
+    if (hand2.toLowerCase() === 'paper') {
+      return 'Hand two wins!';
+    }
+  }
+
+  if (hand1.toLowerCase() === 'paper') {
+    if (hand2.toLowerCase() === 'scissors') {
+      return 'Hand two wins!';
+    }
+    if (hand2.toLowerCase() === 'rock') {
+      return 'Hand one wins!';
+    }
+  }
+
+  if (hand1.toLowerCase() === 'scissors') {
+    if (hand2.toLowerCase() === 'rock') {
+      return 'Hand two wins!';
+    }
+    if (hand2.toLowerCase() === 'paper') {
```

Comment:
Nice job scrubbing your data but you should call the `.toLowerCase()` method only once for each parameter.

You can do so by storing the result of the method to a variable and then comparing it to 'rock', 'scissors', ectc.